docs/content/querying/lookups.md

```diff
@@ -132,6 +132,8 @@ The configuration is propagated to the query serving processes (Broker / Router
 The query serving processes have an internal API for managing lookups on the process and those are used by the Coordinator.
 The Coordinator periodically checks if any of the processes need to load/drop lookups and updates them appropriately.
 
+Please note that only 2 simultaneous lookup configuration propagation requests can be concurrently handled by a single query serving process, to prevent lookup handling from consuming too many server HTTP connections.
```

Comment:
optional: I think this is a very dense sentence, it might be friendlier to break up at the comma, maybe:

> Please note that only 2 simultaneous lookup configuration propagation requests can be concurrently handled by a single query serving process. This is done to prevent lookup handling from consuming too many server HTTP connections.

Thanks for adding it to the docs though :+1: